src/p2p/net_node.inl

```diff
@@ -1350,6 +1350,8 @@ namespace nodetool
   template<class t_payload_net_handler>
   bool node_server<t_payload_net_handler>::try_to_connect_and_handshake_with_new_peer(const epee::net_utils::network_address& na, bool just_take_peerlist, uint64_t last_seen_stamp, PeerType peer_type, uint64_t first_seen_stamp)
   {
+    bool one_conn_below_max = false;
```

Comment:
Why are you messing with the logic in this function?

Reply:
Is this not the perfect place for this logic? try_to_connect_and_handshake_with_new_peer has other logic related to connection counts and conditions for which connections should not be made, which is similar to my additions.